stable/prometheus-blackbox-exporter/README.md

```diff
@@ -70,6 +70,11 @@ The following table lists the configurable parameters of the Blackbox-Exporter c
 | `service.type`                         | type of service to create                       | `ClusterIP`                   |
 | `service.port`                         | port for the blackbox http service              | `9115`                        |
 | `service.externalIPs`                  | list of external ips                            | []                            |
+| `serviceMonitor.enabled`               | If true, a ServiceMonitor CRD is created for a prometheus operator | `false`    |
+| `serviceMonitor.labels`                | Labels for prometheus operator                  | `{}`                          |
+| `serviceMonitor.interval`              | Interval for prometheus operator endpoint       | `30s`                        |
+| `serviceMonitor.url`                   | The URL that blackbox will scrape if serviceMonitor is enabled | `http://www.changemeoriwillfail.com/healthz` |
```

Comment:
Tell me if you see something that is more elegant but still validates with `helm lint`.

Reply:
Oh, this should be `example.com` instead. Saw a funny tweet where someone bought a DNS name and broke things for people.... https://twitter.com/JGamblin/status/1100414626259324929

example.com is a reserved domain. It wont be bought or hijacked. See https://www.iana.org/domains/reserved